dspace-xmlui/src/main/java/org/dspace/app/xmlui/aspect/artifactbrowser/SendItemRequestAction.java

```diff
@@ -118,7 +118,11 @@ public Map act(Redirector redirector, SourceResolver resolver, Map objectModel,
                 )
                 .getRequestItemAuthor(context, item);
 
-        RequestItem requestItem = new RequestItem(item.getID(), Integer.parseInt(bitstreamId), requesterEmail, requesterName, message, Boolean.getBoolean(allFiles));
+	// probably wrong -- Boolean.getBoolean() returns true if and only if
+	// the system property named by the argument exists and is equal to the
+	// string "true".
+        // RequestItem requestItem = new RequestItem(item.getID(), Integer.parseInt(bitstreamId), requesterEmail, requesterName, message, Boolean.getBoolean(allFiles));
```

Comment:
No problem, Bill!  We all had to figure these things out at some point, so we're glad to help you learn some of the tricks with GitHub :)